android/BOINC/app/src/main/java/edu/berkeley/boinc/attach/glide/ScaleBitmapBy2.kt

```diff
@@ -19,13 +19,14 @@
 package edu.berkeley.boinc.attach.glide
 
 import android.graphics.Bitmap
+import androidx.core.graphics.scale
 import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool
 import com.bumptech.glide.load.resource.bitmap.BitmapTransformation
 import java.security.MessageDigest
 
 class ScaleBitmapBy2 : BitmapTransformation() {
```

Comment:
Why do we need this class at all? Is it possible to remove it and update package with better graphics?

Reply:
It's used in Glide to transform the downloaded image before displaying it.